src/polyfills/keyboardevent-key.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Original shim code: https://github.com/shvaikalesh/shim-keyboard-event-key
```

Comment:
Yes this brings IE11's `KeyboardEvent.key` behavior into line with the standard.

Reply:
Why don't we revert? Just because something is deprecated doesn't mean it's going away any time soon. In fact, I'd bet the browsers will not remove it.

And, it'd be smaller. 😄

Reply:
`KeyboardEvent#key` is the recommended property to use, it is not deprecated. `KeyboardEvent#keyCode` [is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode). The problem this polyfill/shim solves is making IE11 behave correctly, because it has nonstandard behavior for `KeyboardEvent#key` since it implemented an early draft of the current spec.

Reply:
Oh you mean revert #18530 ?